### ISSUE TITLE
TGlobal is duplicate (in BlitzMaxNG at least)

### ISSUE THREAD

**Issue by DruggedBunny** [markcwm/openb3d.mod#20]
I just built the latest version in the latest bmx-ng release, but ran into an error with TGlobal, where it said it was a duplicate of that found in pub.reflection (TGlobal is defined from line 1422 of `mod\brl.mod\reflection.mod\reflection.bmx`).

I just renamed all instances in openb3d to `TB3DGlobal` and it built after that -- I renamed the file `mod\openb3dmax.mod\openb3dmax.mod\inc\TGlobal.bmx` to `TB3DGlobal.bmx` and then did a bulk find-in-files using NotePad++ - this was done by simply dragging the whole openb3dmax.mod folder onto NP++, right-clicking the resulting folder tree and choosing Find, then doing the replacement. After that it built fine here.

(Not sure why you don't run into this -- maybe to do with using Framework and specifying imports? I'm just doing `Import openb3dmax.B3dglgraphics` here.)

Thanks for keeping this going, just getting back to it!

**Comment by markcwm** [comment]
Thanks, I do remember this conflict but I don't think I wanted to rename TGlobal as this breaks compatibility with Minib3d a bit, obviously it really has to compile with the latest NG release so I've renamed it to TGlobal3D as it's easier to read. I also had to update the docs and libs repositories. I'm not sure why it errors as I have an older NG (bcc 0.90 w/ gcc 5.4) with TGlobal in BRL.Reflection and there's not even a warning, but must be due to a stricter compiler.
